agents/property.json.graphs_examples

```diff
@@ -2,7 +2,7 @@
     "rte": {
         "predefined_graphs": [
             {
-                "name": "astra_agents",
+                "name": "astra_agents_cn",
```

Comment:
Suggest to use "features" rather than "area" for the name, such as "astra_agents_qwen" might be better

Reply:
agree

Reply:
[restore the previous VSCode extensions](https://github.com/rte-design/ASTRA.ai/pull/147/commits/9a35b1bc8b05eb4598bdfe3ad0c1cc3adcbe0354)